apps/twap/src/app/conditionalOrder/getOrderStatus.ts

```diff
@@ -0,0 +1,68 @@
+import { OrderStatus, OrderStruct } from '../types/order';
+
+/**
+ * Returns if the order is fulfilled.
+ *
+ * @param executedSellAmount Current executed sell amount.
+ * @param orderInfo OrderStruct of the order.
+ * @returns If the order is fulfilled.
+ */
+function isOrderFulfilled(
+  executedSellAmount: bigint,
+  orderInfo: OrderStruct
+): boolean {
+  return (
+    executedSellAmount ===
+    BigInt(orderInfo.partSellAmount) * BigInt(orderInfo.n)
```

Comment:
So this compares `executedSellAmount` of "parent" TWAP order with the `partSellAmount` * `number of parts`. Since each part is executed as a single order, could there be a mismatch between these two amounts even though all parts are executed. For example, can total sell amount of parts be bigger then the `executedSellAmount` due to surplus or something like that? Just asking out of curiosity.